Source/Wikitivity.CustomPage/WikiConstants.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Relativity.Services.Objects;
+
+namespace Wikitivity.CustomPage
+{
+	public class WikiConstants

```

Comment:
Why is this Constants if it's a DTO contract?

Reply:
follow up for clarification